src/network/networkd-link.c

```diff
@@ -199,6 +199,21 @@ static bool link_proxy_arp_enabled(Link *link) {
         return true;
 }
 
+static bool link_ipv6_proxy_ndp_enabled(Link *link) {
```

Comment:
this returns `true` when ipv6 proxy ndp should be enabled for the given link (if enabled by configuration, interface is no loopback interface and we find a network for it).
This was moved into `ipv6_proxy_ndp_is_enabled`.